src/cq_cam/routers.py

```diff
@@ -264,10 +321,17 @@ def shift_polygon(polygon: Path, i: int):
     return polygon
 
 
-def route_polyface_outers(job: "Job", polyfaces: list[PathFace], stepover=None):
+def route_polyface_outers(
+    job: "Job",
+    polyfaces: list[PathFace],
+    stepover=None,
+    start_cv: CommandVector = CommandVector(),
```

Comment:
Setting a default to an instance can lead to unexpected bugs due to mutability. For mutable objects we normally default to None and then create the instance in the function body.